sql/src/main/java/io/crate/planner/operators/RootRelationBoundary.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.planner.operators;
+
+import io.crate.analyze.OrderBy;
+import io.crate.analyze.relations.AbstractTableRelation;
+import io.crate.analyze.symbol.Symbol;
+import io.crate.planner.PlannerContext;
+import io.crate.planner.Merge;
+import io.crate.planner.ExecutionPlan;
+import io.crate.planner.projection.builder.ProjectionBuilder;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+
+public class RootRelationBoundary implements LogicalPlan {
```

Comment:
Maybe also add a short description (e.g. that the primary purpose is to make sure the result is on the handler and no longer distributed)